mscorlib/system/globalization/charunicodeinfo.cs

```diff
@@ -322,7 +338,7 @@ internal static bool IsWhiteSpace(char c)
             }
             return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
 #else
-            return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
+            return EndianSwap(((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
```

Comment:
Cool, I would like to make it a icall if it's not one already. For System z this operation reduces to a single instruction or two.